app/models/application_search_filter.rb

```diff
@@ -1,44 +1,37 @@
+require 'laa_crime_schemas'
+
 class ApplicationSearchFilter
   include ActiveModel::Model
   include ActiveModel::Attributes
 
   DATASTORE_FILTERS = %i[
     search_text
     status
+    review_status
```

Comment:
Introduce `review_status` to filter decided applications (status is not sufficient).